test/extended/testdata/oauth_idp/htpasswd-secret.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: htpasswd
```

Comment:
It might make sense to have the `name` field value as `{name}` so that we can replace it with `strings.Replace` in tests so that the value we're dealing with during cleanups/IdP config does not appear magical.

I don't necessarily insist on this, but it's definitely something to consider.

Reply:
not sure i understand, strings.Replace wouldn't work here, unless i ReadFile/replaceBytes/WriteFile? Maybe i'm missing something.  I've referenced `htpasswdSecretName` to make it less magical.